autogluon/utils/tabular/ml/models/fasttext/hyperparameters/parameters.py

```diff
@@ -0,0 +1,17 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: m.ml.models.fasttext.hyperparameters.parameters.ipynb (unless otherwise specified).
+
+__all__ = ["get_param_baseline"]
+
+# Cell
+# https://fasttext.cc/docs/en/python-module.html#train_supervised-parameters
+def get_param_baseline():
+    params = {
```

Comment:
For max-flexibility, we probably want to include all the parameters here, including those like: 
`dim`, `ws`, etc.

Also those like `thread` that don't affect accuracy but affect speed.